example/demo_survey/src/survey/widgets/visitedPlaces.tsx

```diff
@@ -808,6 +808,23 @@ export const visitedPlaceGeography = {
   geocodingQueryString: function (interview, path) {
     return surveyHelperNew.formatGeocodingQueryStringFromMultipleFields([surveyHelper.get(interview, path, null, '../name')]);
   },
+  invalidGeocodingResultTypes: [
```

Comment:
Should these be in the survey itself? Or should they be part of the map. This is Google maps API specific, so, if it goes in the platform, it should be in some class with Google in its name. Maybe it's easier to put here though, at least for now

Reply:
Yeah, I had the same question myself.
On one hand, it's Google Maps specific, so it could be in evolution directly.
On the other hand, I think it's nice to have the ability in the survey to decide on a per question basis what is considered as "imprecise" or not. Having the types directly in the survey makes this very easy. 

Could leave this for now and see later if we decide to improve this feature